mgmtd/mgmt_main.c

```diff
@@ -154,7 +162,7 @@ const char *zebra_features[] = { "ptm-bfd", NULL };
  */
 const struct frr_yang_module_info zebra_info = {
 	.name = "frr-zebra",
-#if HAVE_BFDD == 0
+#if HAVE_BFDD == 0 || defined(HAVE_RTADV)
```

Comment:
As mentioned above, we don't really need to make this conditional just let the content be a single NULL. Cleaner code, no issue with space/time optimization here.

Reply:
Agreed.